### PR TITLE
disambiguating `Not applicable when using the SDK`

### PR DESCRIPTION
**Pull request by DataWearsAHood**

### CODE REVIEW

File: docs/AppSettings.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Name | Type | Description | Notes
 **useAnimalNames** | **Boolean** | Flag specifying whether animal names should be used for anonymous users. |  [optional]
 **conversationRetentionSeconds** | **Integer** | Number of seconds of inactivity before a conversation’s messages will be automatically deleted. |  [optional]
 **echoPostback** | **Boolean** | A boolean specifying whether a message should be added to the conversation history when a postback button is clicked. |  [optional]
-**ignoreAutoConversationStart** | **Boolean** | A boolean specifying whether a non message event coming from a channel will trigger a start conversation event and count as an active user conversation (AUC). Not applicable when using the SDK. |  [optional]
+**ignoreAutoConversationStart** | **Boolean** | A boolean specifying whether a non message event coming from a social channel will trigger a start conversation event and count as an active user conversation (AUC).<br>**NOTE:** Use of the `startConversation()` or `startConversationWithCompletionHandler` methods from a Web/iOS/Android SDK Client will NOT be blocked by this setting. |  [optional]
```
Comment (Hyjaz) [inline docs/AppSettings.md:11]: `startConversationWithCompletionHandler()`

Comment (Hyjaz) [reply docs/AppSettings.md:11]: Does `Client` need to be capitalized?

Comment (Hyjaz) [reply docs/AppSettings.md:11]: Do we also need to capitalize NOT?

Comment (Mario54) [reply docs/AppSettings.md:11]: I think we can omit `Client` altogether. I also think not should not be capitalized.

**Comment by DataWearsAHood** [comment]
superceded by https://github.com/smooch/smooch-api-specification/pull/108

**Pull request closed without merging**